sdk/src/main/java/io/dapr/client/DaprClient.java

```diff
@@ -525,7 +540,9 @@
    * @param request Request to fetch secret.
    * @return Key-value pairs for the secret.
    */
-  Mono<Response<Map<String, String>>> getSecret(GetSecretRequest request);
+  default Mono<Map<String, String>> getSecret(GetSecretRequest request) {
+    throw new UnsupportedOperationException();
```

Comment:
What is the reason to provide default implementations?

Reply:
I think it's generally a good idea to add default impl when adding new methods to an interface so that existing implementation won't break.

Reply:
OK. So this basically moves a compilation error to runtime error. Compilation errors are way easier to catch. If a class does not implement this, we should know right away. I've had many refactoring tasks that were made easier because of compilation errors. Please, remove default implementations. This is preference. Nothing wrong with your approach.

Reply:
done